Cargo.toml

```
@@ -42,7 +42,7 @@ description = "A runtime for writing automotive and industrial Rust on top of Hi
 
 [workspace.dependencies]
 bitflags = "2.4"
-defmt = "=0.3.6"
+defmt = ">=0.3.8"
```

Why do we do this? Basically this promises that this will work on any future version of `defmt`, even a hypothetical `1.0.0` that would break compatibility.

Why not:

```suggestion
defmt = "0.3"
```

---

Good point, I'll fix this :+1:

---

We need it to be `0.3.8` or higher because `0.3.7` breaks our build. Technically, it can also be `<0.3.7`, but I think it makes sense to limit the dependency to the fixed version and higher.